jplag.frontend.golang/src/main/java/de/jplag/golang/grammar/GoTokenUtils.java

```diff
@@ -0,0 +1,21 @@
+package de.jplag.golang.grammar;
+
+import de.jplag.golang.GoToken;
+
+public class GoTokenUtils {
+
+    private static final int NOT_SET = -1;
+    private static final String NOT_SET_STRING = "";
+
+    public static GoToken getDummyToken(int type) {
+        return new GoToken(type, NOT_SET_STRING, NOT_SET, NOT_SET, NOT_SET);
+    }
+
+    public static GoToken getDummyToken(int type, String fileName) {
+        return new GoToken(type, fileName, NOT_SET, NOT_SET, NOT_SET);
+    }
+
+    private GoTokenUtils() {
+        // Utility class, should not be instantiated.
+    }
```

Comment:
the constuctor should rather be at the beginning of the class functions even if it is empty or private